src/cloud_provider/aws/databases/mongodb.rs

```diff
@@ -5,8 +5,9 @@ use tera::Context as TeraContext;
 use crate::cloud_provider::environment::Kind;
 use crate::cloud_provider::service::{
     check_service_version, default_tera_context, delete_stateful_service, deploy_stateful_service, get_tfstate_name,
-    get_tfstate_suffix, send_progress_on_long_task, Action, Backup, Create, Database, DatabaseOptions, DatabaseType,
-    Delete, Downgrade, Helm, Pause, Service, ServiceType, StatefulService, Terraform, Upgrade,
+    get_tfstate_suffix, scale_down_database, send_progress_on_long_task, Action, Backup, Create, Database,
+    DatabaseOptions, DatabaseType, Delete, Downgrade, Helm, Pause, Service, ServiceType, StatefulService, Terraform,
+    Upgrade,
 };
```

Comment:
OPT: question, maybe it starts to be a lot of individual imports? I am not sure how bad would importing `use crate::cloud_provider::service;` and use `service::get_tfstate_suffix` for example.
That's really a question, I have no idea if there are any downside.